### ISSUE TITLE
Unknown command '__fish_sgrep'

### ISSUE THREAD

**Issue by nwykes** [fish-shell/fish-shell#2372]
Using https://github.com/fish-shell/fish-shell/commit/b85a8bbbfed56f603dca5d6da85858f9c973b669 gives the following when trying to complete:

```fish
 cd .cofish: Unknown command '__fish_sgrep'
/usr/local/Cellar/fish/HEAD/share/fish/functions/__fish_complete_cd.fish (line 20): 	if echo $ctoken | __fish_sgrep '^/\|^\./\|^\.\./\|^~/' >/dev/null
                                                                                    	                  ^
in function '__fish_complete_cd'
	called on standard input

in command substitution
	called on standard input
```

**Comment by bucaran** [comment]
Must have something to do with this. 

#2324

/cc @faho

**Comment by faho** [comment]
Oh crap, I did a `git commit -a` when I fixed up #2324 for merge and it didn't catch the rename. Sorry, should be fixed now.

**Comment by nwykes** [comment]
Confirmed. Thanks!
